README.md

```diff
@@ -289,6 +291,9 @@ To fix this, do the following:
 * Change the locale by Windows Regional Settings to United States and English on the server and all hudson nodes. After that tf.exe should output dates in english, which can be parsed properly.
 * Start Hudson using the UnitedStates, English locale. Either set it using `-Duser.language=en -Duser.country=US` on the command line or check the documentation for the container that Hudson is running within.
 
+### If I have multiple artifacts linked in my Release Definition, will this plugin trigger a release?** <br>
+    Yes, it is supported from 1.3 version onwards.
```

Comment:
Can you remove the leading spaces?  It's currently causing the answer to be formatted like source code.